docs/src/pages/guides/interoperability/interoperability.md

```diff
@@ -386,7 +386,33 @@ const CustomizedSlider = styled(Slider)(
 
 ### Portals
 
-TODO: fill this section after the portal is implemented with the new styled engine.
+The [Portal](/components/portal/) provides a first-class way to render children into a DOM node that exists outside the DOM hierarchy of the parent component.
+Because of the way styled-components scopes its CSS, you may run into issues where styling is not applied.
+
+For example, if you attempt to style the [Menu](/components/menus/) of a [Select](/components/selects/) component using the property `MenuProps`,
+you will need to pass along the `className` property to the element being rendered outside of it's DOM hierarchy.
+The following example shows a workaround:
+
+```jsx
+import React from 'react';
+import { experimentalStyled as styled } from '@material-ui/core/styles';
+import Menu from '@material-ui/core/Menu';
+import MenuItem from '@material-ui/core/MenuItem';
+
+const StyledMenu = styled(({ className, ...props }) => (
+  <Menu {...props} classes={{ paper: className }} />
+))`
+  box-shadow: none;
+  border: 1px solid #d3d4d5;
+
+  li {
+    padding-top: 8px;
+    padding-bottom: 8px;
+  }
+`;
+```
+
+{{"demo": "pages/guides/interoperability/StyledComponentsPortal.js"}}
```

Comment:
We have, more or less, the same demo in https://next.material-ui.com/components/menus/#customized-menu. Did you consider linking it instead? The demo looks better but would need to be migrated to the styled API. 

We have never done this before, but its technically possible:

```suggestion
{{"demo": "pages/components/menus/CustomizedMenus.js"}}
```

Reply:
I haven't noticed that we have similar demo, it makes sense to reuse it 👍 Will update that one to use the `styled()` API

Reply:
I saw now that this demo is completely broken, will fix it and link that one on the interoperability guide

Reply:
Ok, so the demo is working as expected, but it is not a good showcase for how the portal can be customized, as the root of the Menu is the Popover component. What other component may be a good candidate for it?

Reply:
Oh right, I see. For the branding of the pricing page, I had to customize the label of the Tooltip. I did so by wrapping the Popper and providing it as a prop, but I think that the classes approach would be better for this showcase.

Reply:
Alright, that would be a good use-case, let me adjust the example to use the `classes.popper` on the `Tooltip` component

Reply:
Updated using the `Tooltip` component